packages/example-getting-started/docs/4-todo-model.md

```diff
@@ -0,0 +1,104 @@
+### Building the Todo model
+
+The Todo model will be the object we use both as a Data Transfer Object (DTO) on
```

Comment:
A link to what a Data Transfer Object might be helpful, unless we assume users know what this is (I don't 😆 ).